Revert "trlte-common: sepolicy: Clean-up policy for external sdcards"

ripee opened the pull request:
Without reverting commit 5b0d29d neither tblte nor trlte will build.

* Yes, this looks horrendously wide-open, but this only applied for the
  complete sandbox that is external sdcard

Change-Id: Ibd1fe240eeed65f079e810a3da5157a4e64944f2 (reverted from commit 5b0d29def8c34686ea540fd315f33b6d7bda3498)

Commit:
trlte-common: sepolicy: Clean-up policy for external sdcards

* Yes, this looks horrendously wide-open, but this only applied for the
  complete sandbox that is external sdcard

Change-Id: Ibd1fe240eeed65f079e810a3da5157a4e64944f2 (reverted from commit 5b0d29def8c34686ea540fd315f33b6d7bda3498)

Commit:
Compress recovery ramdisk with LZMA

This allows for the proper making of the recovery image.

micky387 commented:
otherwise, to do things right ,you need to revert this commit (git revert 5b0d29def8c34686ea540fd315f33b6d7bda3498)
instead of removing it yourself
EDIT: you are true,with repo sync yesterday,no build error.
Today,failed. So needed for build success

github-cygwin reviewed (commented):
What about using

  https://review.lineageos.org/#/c/208346

instead of LZMA_RAMDISK_TARGETS? Does that help?

Not that LZMA_RAMDISK_TARGETS is a bad idea, but it might be unnecessary as
soon as the above patch is merged.

EDIT: This fixed the recovery size issue for me on kccat6, which has the smallest
recovery partition size of all our devices.

Commit:
Compress recovery ramdisk with LZMA

This allows for the proper making of the recovery image. (reverted from commit 36b3f347c04f3526608fcacf3bc4477b2fa10589)

ripee commented:
I reverted LZMA_RAMDISK_TARGETS out of this pull request and I'm building trlte against right now.

Edit: Ok, it built with a slightly reduced final zip file size. Now let's see what the users have to say.

micky387 commented:
ripee ,it does not affect users....
corinna, yes,we can remove this flag,not needed .you or ripee can close this pull request and a add new with good commit revert

github-cygwin commented:
I pushed the revert for commit 5b0d29def8c3

micky387 commented:
You have pushed to the wrong branch.
regards

ripee commented:
Yup, it's in cm-14.1.

Can we make the lineage-15.1 branch the default in each repo that has one, for convenience?

github-cygwin commented:
On Mar 18 10:25, micky387 wrote:
> You're pushed to the wrong branch.
> regards

Fixed.


Thanks,
Corinna

github-cygwin commented:
On Mar 18 10:49, ripee wrote:
> Yup, it's in cm-14.1.
> 
> Can we make the lineage-15.1 branch the default in each repo that has
> one, for convenience?

Can you please come over into IRC #apq8084 and *stay* there, so there's
a chance to communicate even when living in different timezones?


Thanks,
Corinna
